### PR TITLE
Header adjustments

### LINKED ISSUE #40: Insert Handbuch into header
Insert link of Handbuch into header.
Insert Nutzerin instead of account.
(Solves #13 -> Deleting header)

### LINKED ISSUE #46: Replace resources for handbook with get route

### PR DESCRIPTION
**Pull request by lauralindal**
Resolves #40 

Resolves #46

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,21 +1,21 @@
 Rails.application.routes.draw do
 
-  
+
```
Comment (schultyy) [inline config/routes.rb]: Whitespace

File: config/routes.rb [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
   resources :institutions
   resources :users
   resources :occasions
-  resources :handbook
 
   root 'static_pages#home'
 
   get 'sessions/new'
-  
+
```
Comment (schultyy) [inline config/routes.rb]: Whitespace

File: config/routes.rb [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
   get    'login'   => 'sessions#new'
   post   'login'   => 'sessions#create'
   delete 'logout'  => 'sessions#destroy'
- 
+
```
Comment (schultyy) [inline config/routes.rb]: Whitespace

File: config/routes.rb (hunk 1 above)
Comment (noobiba) [inline config/routes.rb]: Ignored.

File: config/routes.rb (hunk 2 above)
Comment (noobiba) [inline config/routes.rb]: Ignored.

File: config/routes.rb (hunk 3 above)
Comment (noobiba) [inline config/routes.rb]: Ignored.

**Pull request merged**